modules/silicon_design/main.tf

```diff
@@ -298,7 +298,9 @@ resource "google_notebooks_instance" "ai_notebook" {
   metadata = {
     terraform  = "true"
     proxy-mode = "service_account"
```

Comment:
Can you add a comment about how it interact with post_startup_script?

Reply:
I removed this one bit altogether as it didn't seem essential to this pull request, we might want to revisit it at a later time.